src/responses.ts

```diff
@@ -14,10 +14,22 @@ export type Alias = string
  * Either an alias of a member or their blockchain address
  */
 export type AddressOrAlias = Address | Alias
+
 /**
- * A member including their blockchain address as well as their local alias
+ * A role for a member
  */
+export type Role = string | undefined
+
+/**
+ * A body for a member
+ */
+export type MemberBody = {
+  alias: Alias
```

Comment:
`alias` should also be optional as we're doing a `patch`. You need to update the logic of the method to only set the alias if it is passed